confluent_kafka/avro/error.py

```diff
@@ -20,7 +20,10 @@ class ClientError(Exception):
     """ Error thrown by Schema Registry clients """
 
     def __init__(self, message, http_code=None):
-        self.message = message
+        if http_code:
+            self.message = '{}. Error code: {}'.format(message, str(http_code))
+        else:
+            self.message = message
```

Comment:
Not sure which is more preferred, but this could also be written as

```python
self.message = message
if http_code:
    self.message += ". Error code: {}".format(code)
```